python/composio/tools/local/mathematical/actions/calculator.py

```diff
@@ -19,12 +21,54 @@ class CalculatorResponse(BaseModel):
 
 class Calculator(LocalAction[CalculatorRequest, CalculatorResponse]):
     """
-    Useful to perform any mathematical calculations, like sum, minus, multiplication, division, etc.
+    Performs mathematical calculations such as addition, subtraction, multiplication, and division.
```

Comment:
Consider expanding the class docstring to include information about supported operations and any limitations of the calculator. This would provide better documentation for users of the Calculator class.